.github/ISSUE_TEMPLATE/suggest-feature.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "💡 Suggest an Improvement or Feature"
+about: Suggest an enhancement or a new feature for the Portmaster
+labels: feature
+---
+
+**What would you like to add or change?**:
+
+
+
+**Why do you and others need this?**:
```

Comment:
👌 - the only thing I would change here is add two additional newlines at the end of the file